### DISCUSSION TITLE
Are there any upcoming releases planned for "Fabrication"?

### DISCUSSION THREAD

**Discussion by RUTSTER** [MrTJP/ProjectRed#1797] [Q&A] [answered]
I've had the opportunity to test the ["Fabrication v2: Alpha Release"](https://github.com/MrTJP/ProjectRed/discussions/1730) version, and I must say it's fantastic! 
I'm eager to touch other planned features)

However, I've noticed that there hasn't been any further development on "Fabrication v2" since 2022. 
Could you please clarify if this is a temporary pause in "Fabrication" development, or if the decision has been made to cancel the project altogether?

**Comment by MrTJP** [comment, marked as answer]
It's in active development, but time is split with many other features in the mod. I will typically move from one thing to another, so temporary pauses are expected.
